internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -42,7 +50,10 @@ func (s *Server) SetUpSession(stream s2av2pb.S2AService_SetUpSessionServer) erro
 		case *s2av2pb.SessionReq_OffloadResumptionKeyOperationReq:
 			// TODO(rmehta19): Implement fake.
 		case *s2av2pb.SessionReq_ValidatePeerCertificateChainReq:
-			// TODO(rmehta19): Implement fake.
+			if resp, err = validatePeerCertificateChain(req.GetValidatePeerCertificateChainReq()); err != nil {
```

Comment:
nit: s/resp/_

Since `resp` is unused at present.

Reply:
I think resp is used, since I send the resp after the switch statement, I declared it above. Regardless, I split up the call to the helper function and checking of the error into two separate lines, thanks!